imgr.py

```diff
@@ -221,7 +225,7 @@ def move(hutch, ioc, hostport):
             hp = hostport.split(":")
             c['newhost'] = hp[0]
             if len(hp) > 1:
-                c['newport'] = int(hp[1])
+                c['newport'] = port_to_int(hp[1], hp[0], cl)
```

Comment:
To double-check my understanding: this means that when we move an IOC from one host to another using imgr, it will be able to select a new port that doesn't conflict with the existing port usage on the new host, rather than simply re-using the existing port setting from the previous host?

Reply:
Yes, if we use the keywords "open" or "closed".